CHANGELOG.md

```diff
@@ -8,10 +8,19 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.5.7] - 2024-07-25
+
 ### Added
 
 * Support requesting a different consistency option per request in `Check`, `Expand`, `ListObjects`, `ListUsers`, and `Read` [#1764](https://github.com/openfga/openfga/pull/1764)
+  * This is currently experimental and needs to be enabled by configuring `OPENFGA_EXPERIMENTALS=enable-consistency-params` or passing `--experimentals enable-consistency-params` to `openfga run`.
   * When `HIGHER_CONSISTENCY` is requested, OpenFGA will skip the check resolver cache. For storage implementors it is recommended to skip any caching and perform a stronger read if `HIGHER_CONSISTENCY` is requested. This can be accessed in the `Consistency` options provided to the relevant methods of the storage interface.
+* Start publishing images to `ghcr.io/openfga/openfga` as alternative to DockerHub [#1775](https://github.com/openfga/openfga/pull/1775) - Thanks @JAORMX!
+* Performance improvements for parent child relations in Check [#1765](https://github.com/openfga/openfga/pull/1765)
```

Comment:
Please also add https://github.com/openfga/openfga/pull/1786:

> Performance improvement in Check: computed relations don't consume from the resolution depth quota, doesn't trigger additional goroutines, and doesn't get cached.

Reply:
Added in #6a93ae97e171dc17c1670a4f648508c1c77b736d